From f255ef2e16510d1ae6d09bb004dabf7ce67788e7 Mon Sep 17 00:00:00 2001
From: linyingxiong <yingxiong.lin@quvideo.com>
Date: Mon, 1 Sep 2025 12:49:10 +0800
Subject: [PATCH] always enable cmake install

---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6ae6955..3ab0f1e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -156,7 +156,6 @@ if(BUILD_STATIC_LIBS)
 	export(TARGETS efsw-static NAMESPACE efsw:: APPEND FILE ${CMAKE_CURRENT_BINARY_DIR}/cmake/${PROJECT_NAME}Targets.cmake)
 endif()
 
-if(EFSW_INSTALL)
 	install(TARGETS efsw EXPORT efswExport
 		LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
 		ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
@@ -179,7 +178,6 @@ if(EFSW_INSTALL)
 
 	install(EXPORT efswExport NAMESPACE efsw:: DESTINATION "${packageDestDir}" FILE ${PROJECT_NAME}Targets.cmake)
 	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cmake/efswConfig.cmake DESTINATION "${packageDestDir}")
-endif()
 
 if(BUILD_TEST_APP)
 	# C++ test application
-- 
2.44.0.windows.1

